server/models/user.js

```diff
@@ -22,6 +40,7 @@ const userSchema = new Schema({
   github: { type: String },
   email: { type: String, unique: true },
   tokens: Array,
```

Comment:
@catarak I was thinking about renaming `apiKeys` on the line below to `accessTokens` but then saw this `tokens` array. It's where we store the [accessTokens from Github/Google](https://github.com/processing/p5.js-web-editor/blob/master/server/config/passport.js#L89) when a user does a social login. 

Do you know what we use these for? I guess it's a good idea to keep the naming distinct to avoid confusion? I briefly thought that we might put the Personal Access Tokens in this array too, but maybe that's a bad idea.

Reply:
I went back to the code I used as reference for setting up passport (https://github.com/sahat/hackathon-starter), and I think the tokens are supposed to be used for linking/unlinking social accounts. So, maybe, this access token should be put into that array? I don't know. I think the linking/unlinking social accounts isn't fully fleshed out anyway.

Reply:
🤔 

So the `tokens` are keys the app has been given by external services. And `apiKeys` will be keys that the app has given to others. In which case, perhaps they should be separate.

Reply:
that makes sense to me!